src/component/reducer/Reducer.ts

```diff
@@ -85,6 +85,7 @@ export const getInitialState = (): State => ({
   activeSpectrum: null,
   mode: 'RTL',
   molecules: [],
+  floatingMolecules: [],
```

Comment:
@stropitek I think this property should not be at the first level but rather in another prop 'view' or something like that. WDYT ?

Reply:
Let's but in a `view` property, it will help identify what has already been identified as not belonging to the core data state.